src/ensembl/src/shared/hooks/useOutsideClick.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+
+export default function useOutsideClick<T extends HTMLElement>(
+  ref: React.RefObject<T>,
+  callback: () => void
+): void {
```

Comment:
`void` as return type is not necessary here

Reply:
Removed it